slsDetectorSoftware/src/CmdProxy.cpp

```diff
@@ -1,4 +1,5 @@
 #include "CmdProxy.h"
+#include "bit_stuff.h"
```

Comment:
you want to rename this from bit_stuff.h to maybe bit_manupulation.h or something similar?